redaxo/src/addons/install/lang/en_gb.lang

```diff
@@ -58,6 +58,7 @@ install_to_addon_page = Go to AddOn management
 
 install_settings_general = General
 install_settings_backups = Create backup
+install_settings_backups_note = Before updating a package, the installer saves a copy of the previously installed version in <code>/redaxo/data/addons/install/</code>. This copy can be restored in case of errors.
```

Comment:
```suggestion
install_settings_backups_note = Before updating a package, the installer saves a copy of the previously installed version in <code>redaxo/data/addons/install/</code>. This copy can be restored in case of errors.
```

?